client/blocks/comments/style.scss

```diff
@@ -350,3 +354,12 @@
 		transform: rotate( 180deg );
 	}
 }
+
+.comments__comment-count {
+	color: lighten( $gray, 10% );
+	size: 13px;
```

Comment:
Is this supposed to be `font-size`? I think `13px` looks much better though. The original `17px` is a bit much in my opinion.